docs/install/tutorials/rpi.rst

```diff
@@ -300,9 +312,9 @@ Installing Kolibri
 Set up Kolibri local domain
 ***************************
 
-After completing the installation, you can make kolibri available on port ``:80`` in addition to ``:8080``. This will make it possible to type, for example, a domain ``kolibri.lan`` in the browser location bar, and because of our captive portal, it will display.
+After completing the installation, you can make kolibri available on port ``:80`` in addition to ``:8080``. This will make it possible to type, for example, a domain ``kolibri.library`` in the browser location bar, and because of our captive portal, it will display.
```

Comment:
For consistency with what's shown in the screenshot further up.

Reply:
I can create a new screenshot instead, IIRC we preferred `.library` in an earlier discussion.